scipy/linalg/tests/test_lapack.py

```diff
@@ -27,6 +27,10 @@
 from scipy.linalg.lapack import get_lapack_funcs
 from scipy.linalg.blas import get_blas_funcs
 
+# Special import for monkey patched functions
+from scipy.linalg.lapack import dsyevr, ssyevr, cheevr, zheevr
```

Comment:
The change in lapack.py needs to be made so that this is not necessary.
In particular, `get_lapack_funcs` should return the same functions.